src/datastore/postgres-store.ts

```diff
@@ -4230,6 +4231,28 @@ export class PgDataStore
     });
   }
 
+  async getTxsCount({ from, to }: { from: number; to: number }) {
+    let resultQuery: QueryResult<TransactionsCount>;
+    return this.queryTx(async client => {
+      const maxHeight = await this.getMaxBlockHeight(client, { includeUnanchored: true });
+      resultQuery = await client.query<TransactionsCount>(
+        `
+          SELECT DATE_TRUNC('day', to_timestamp(parent_burn_block_time)) as "day", count(*)::int as count
+          FROM txs
+          WHERE canonical = true AND 
+          microblock_canonical = true AND 
+          block_height <= $1 AND 
+          parent_burn_block_time >= $2 AND 
+          parent_burn_block_time <= $3
```

Comment:
These dates are all stored in UTC. Will you need to translate dates to user's local times for explorer display?